### ISSUE TITLE
Localizer: error: col_name not found:

### ISSUE THREAD

**Issue by briochh** [jtwhite79/pestpp#8]
I appear to be running into an issue after the first ensemble run through unsing `pestpp-ies` at the stage:
`preparing fast-look containers for threaded localization solve`

The error being:
```
Error condition prevents further execution:
Localizer::get_localizing_obs_hadamard_matrix() error: col_name not found: DRNCOND_000
```
So it looks like an error in the localizer set up somewhere possibily with mis-matched cases? all my paramters are lowercase.

My localizer matrix is a pretty rudimentary 0 or 1 localising transport pars from flow obs, by group (maybe also a problem?) (see attched).

[localizer.mat.txt](https://github.com/jtwhite79/pestpp/files/2359754/localizer.mat.txt)

**Comment by jtwhite79** [comment]
Looks like the instantiation of the localizer is getting confused - its trying to call ``Localizer::get_localizing_obs_hadamard_matrix()`` with a single par name (a par name not listed explicitly in the localizing matrix).  files?

**Comment by jtwhite79** [comment]
this is fixed as of ``4.1.8``  - I tested it on the files you sent using both ``ies_localize_how(obs)`` and ``ies_localize_how(pars)`` and seems to be working.

**Comment by briochh** [comment]
Thanks Jeremy!
